src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -1833,7 +1833,8 @@ private string GetDefaultValue(Node nd, Field field)
 
             if (IsOptional(field) || IsAnyList(field.Type))
             {
-                return string.Format("default({0})", GetRedPropertyType(field));
+                var type = GetRedPropertyType(field);
+                return type == "SyntaxTokenList" ? "default(SyntaxTokenList)" : "default";
```

Comment:
I'm actually not a fan of this change. There are lots of lines there's just a bunch of typeless `default` literals, which is even less info than there is now.

Reply:
Hrmm... i could return it.  But i didn't think there was a lot of value in those 'defaults'.

Reply:
An alternative would be using named parameters, which I would be fine with.